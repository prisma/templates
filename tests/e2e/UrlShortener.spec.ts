import { testTemplate } from './__testers__'

testTemplate({
  templateName: 'UrlShortener',
  expectedDevOutput: /Top users \(alphabetical\):/,
})
