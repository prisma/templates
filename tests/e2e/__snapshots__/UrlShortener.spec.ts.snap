// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UrlShortener: dev stderr 1`] = `""`;

exports[`UrlShortener: init stderr 1`] = `""`;

exports[`UrlShortener: init stdout 1`] = `
"
> init
> prisma db push && prisma db seed

Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource \\"db\\": PostgreSQL database \\"url_shortener\\", schema \\"public\\" at \\"localhost:5401\\"

PostgreSQL database url_shortener created at localhost:5401

ðŸš€  Your database is now in sync with your schema. Done in XXXms

Running generate... (Use --skip-generate to skip the generators)
Running generate... - Prisma Client
âœ” Generated Prisma Client (3.12.0 | library) to ./node_modules/@prisma/client in
 XXXms

Environment variables loaded from .env
Running seed command \`ts-node prisma/seed\` ...

ðŸŒ±  The seed command has been executed."
`;
